src/celeritas/ext/GeantImporter.cc

```diff
@@ -481,6 +485,7 @@ GeantImporter::GeantImporter(GeantSetup&& setup) : setup_(std::move(setup))
 ImportData GeantImporter::operator()(const DataSelection& selected)
 {
     ImportData import_data;
+
```

Comment:
Unless my github diff viewer is broken, it looks like you accidentally reverted the actual importing of the data in here? `AllElementReader` isn't used, nor are the SB/livermore readers.